packages/smooth_app/lib/pages/product/common/product_list_page.dart

```diff
@@ -323,6 +323,8 @@ class _ProductListPageState extends State<ProductListPage>
             setState(() => barcodes.removeAt(index));
           }
           //ignore: use_build_context_synchronously
```

Comment:
I think we can get rid of this `//ignore: use_build_context_synchronously`

like 
```dart
if(!mounted){
    return;
}
```

Reply:
Done ! Thanks @AshAman999